pkg/config/config.go

```diff
@@ -229,7 +229,7 @@ func (c *Config) GetStepConfig(flagValues map[string]interface{}, paramJSON stri
 
 	if verbose, ok := stepConfig.Config["verbose"].(bool); ok && verbose {
 		log.SetVerbose(verbose)
-	} else if !ok {
+	} else if !ok && stepConfig.Config["verbose"] != nil {
 		log.Entry().Warnf("invalid value for parameter verbose: '%v'", stepConfig.Config["verbose"])
```

Comment:
How about a hint that tells the user how verbose should be defined?

Reply:
... other PR I would say. The scope of this PR is only to not emit a value in case there is no verbose flag.